src/parameterizations/vertical/MOM_energetic_PBL.F90

```diff
@@ -598,6 +599,7 @@ subroutine energetic_PBL(h_3d, u_3d, v_3d, tv, fluxes, dt, Kd_int, G, GV, CS, &
 
 
       U_Star = fluxes%ustar(i,j)
+      U_Star_Mean = sqrt(sqrt(fluxes%taux(i,j)**2+fluxes%tauy(i,j)**2)/GV%rho0)
```

Comment:
The quantities `fluxes%taux` and `fluxes%tauy` are staggered on an Arakawa C-grid w.r.t. to where `fluxes%ustar` lives (cell center).

[L647](https://github.com/NOAA-GFDL/MOM6/blob/dev/master/config_src/coupled_driver/MOM_surface_forcing.F90#L647), [L672](https://github.com/NOAA-GFDL/MOM6/blob/dev/master/config_src/coupled_driver/MOM_surface_forcing.F90#L672-L673), [L693](https://github.com/NOAA-GFDL/MOM6/blob/dev/master/config_src/coupled_driver/MOM_surface_forcing.F90#L693) and [L695](https://github.com/NOAA-GFDL/MOM6/blob/dev/master/config_src/coupled_driver/MOM_surface_forcing.F90#L695) each have an expression of the form
> u* = sqrt( 1/rho * ( gust + |tau| ) )

but I don't see any way to reliably work out from u* what the |tau| contribution was. Either store |tau| in `fluxes` or approximate `U_Star_Mean` as
```fortran
U_Star_Mean = sqrt( 0.5*sqrt( (fluxes%taux(I-1,j) + fluxes%taux(I,j))**2 &
                             +(fluxes%tauy(i,J-1) + fluxes%tauy(i,J))**2 )/GV%rho0)
```
(and check my math).